packages/editor-ui/src/components/NodeDetailsView.vue

```diff
@@ -161,10 +161,19 @@ export default mixins(
 			triggerWaitingWarningEnabled: false,
 			isDragging: false,
 			mainPanelPosition: 0,
+			eventBus: dataPinningEventBus,
```

Comment:
We should avoid storing a Vue component inside of `data` due to circular dependencies.

Instead, you can use `dataPinningEventBus.$on` and `dataPinningEventBus.$off`directly.

Reply:
Thank you! I wasn't aware of this.